src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<blockquote>There is also the ExternalCodeComp type of component, which allows...</blockquote><p> </p><p>I would suggest: "Among the more specific&nbsp;<a href="https://openmdao.org/newdocs/versions/latest/features/building_blocks/components/components.html" target="_blank">components</a>&nbsp;OpenMDAO provides, the&nbsp;<a href="https://openmdao.org/newdocs/versions/latest/features/building_blocks/components/external_code_comp.html" target="_blank">ExternalCodeComp</a>&nbsp;is a specialized ExplicitComponent which allows..."</p><p> </p><blockquote>When applicable the parallel will be made with the pure_python definition of the component, this way the explanation will be clearer.</blockquote><p>--&gt; "When applicable<strong>,</strong> the parallel ..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Changed in f33eb885